cime_config/testdefs/testlist_clm.xml

```diff
@@ -3636,6 +3636,27 @@
     </options>
   </test>
 
+  <test name="RXCROPMATURITY_Lm61" grid="f10_f10_mg37" compset="IHistClm50BgcCrop"  testmods="clm/cropMonthOutput">
+    <machines>
+      <machine name="derecho" compiler="intel" category="rxcropmaturity"/>
```

Comment:
Add a bit about the new rxcropmaturity test list to the top of the file.

Reply:
And we should still run this as part of the ctsm_sci test list also correct?

Reply:
> Add a bit about the new rxcropmaturity test list to the top of the file.

Will do.

> And we should still run this as part of the ctsm_sci test list also correct?

This one no, because it's the coarse-resolution version. The 1-degree version is still in ctsm_sci.

Reply:
Resolved in 752ba73.